cohere/responses/custom_model.py

```diff
@@ -79,8 +79,11 @@ def from_dict(cls, data: Dict[str, Any]) -> "CustomModel":
 class ModelMetric(CohereObject):
     created_at: datetime
     step_num: int
-    loss: float
-    accuracy: float
+    loss: Optional[float] = None
+    accuracy: Optional[float] = None
+    f1: Optional[float] = None
+    precision: Optional[float] = None
+    recall: Optional[float] = None
```

Comment:
ah sry, I went off a curl of a generative request, totally forgot about this. agree w the optional!